accessibility-checker-engine/src/v2/checker/accessibility/util/legacy.ts

```diff
@@ -419,6 +420,165 @@ export class RPTUtil {
         }
     }
 
+    /**
+     * a target is en element that accept a pointer action (click or touch)
+     * 
+     */
+    public static isTarget(element) {
+        if (!element) return false;
+        
+        if (element.hasAttribute("tabindex") || RPTUtil.isTabbable(element)) return true;
+        
+        const roles = RPTUtil.getRoles(element, true); 
+        if (!roles && roles.length === 0)
+            return false;
+
+        let tagProperty = RPTUtil.getElementAriaProperty(element);
+        let allowedRoles = RPTUtil.getAllowedAriaRoles(element, tagProperty);
+        if (!allowedRoles && allowedRoles.length === 0)
```

Comment:
This will always be false. Is this supposed to be `if (!allowedRoles || allowedRoles.length === 0)` ?

Reply:
changed